basic-application/basic-application-core/src/main/java/fr/openwide/core/basicapp/core/business/referential/model/search/IAbstractGenericListItemSearchQuery.java

```diff
@@ -0,0 +1,11 @@
+package fr.openwide.core.basicapp.core.business.referential.model.search;
+
+import fr.openwide.core.jpa.more.business.generic.model.EnabledFilter;
+import fr.openwide.core.jpa.more.business.generic.model.GenericListItem;
+import fr.openwide.core.jpa.more.business.search.query.ISearchQuery;
+import fr.openwide.core.jpa.more.business.sort.ISort;
+
+public interface IAbstractGenericListItemSearchQuery<T extends GenericListItem<? super T>, S extends ISort<?>> extends ISearchQuery<T, S> {
```

Comment:
This should be renamed `IGenericListItemSearchQuery`: there's no point in describing an interface as `abstract` (an interface is *always* abstract).